styles/globals.css

```diff
@@ -1,16 +1,58 @@
-html,
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Beau+Rivage&family=Noto+Serif&family=Shadows+Into+Light&family=Square+Peg&family=Whisper&display=swap");
+
 body {
-  padding: 0;
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+  background-image: url("../pictures/background.jpg");
+  color: rgb(102, 172, 166);
+  font-family: "Playfair Display";
+}
+
+.content {
+  margin: 10px auto;
 }
 
-a {
-  color: inherit;
+nav {
+  margin: 10px auto 80px;
+  padding: 10px 0;
+  box-shadow: 1px 1px 5px rgb(110, 160, 161);
+  border-bottom: 1px solid rgb(110, 160, 161);
+}
+
+nav .logo {
+  margin-right: 10px;
+  margin-top: -60px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: "Beau Rivage";
+  font-size: 70px;
+  text-shadow: 1px 1px 2px rgb(9, 51, 63), 0 0 1em;
+}
+
+.links {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.links a{
+  font-size: 25px;
+  color: rgb(144, 243, 221);
+  text-shadow: 1px 1px 2px rgb(15, 37, 37), 0 0 1em;
   text-decoration: none;
 }
 
-* {
-  box-sizing: border-box;
+.links a:hover{
+  color: rgb(20, 245, 196);
+  text-shadow: 1px 1px 2px rgb(159, 252, 232), 0 0 2em;  
+}
+
+footer{
+  margin-top: 60px;
+  padding: 30px 0;
+  display: block;
+  text-align: center;
+  font-size: 20px;
+  color: rgb(144, 243, 221);
+  border-top: 1px solid rgb(110, 160, 161);
+  box-shadow: 1px 1px 5px rgb(110, 160, 161);
```

Comment:
Styling the components could be done with modules. Each component has its own css (or scss if we use SASS) file for separation of concern. Leaving globals.css for CSS that styles site global elements.
_ex Navbar imports styles from Navbar.modules.css_

Reply:
Great reflection. Fixed it as requested, on the "Nav" incase someone wants to use Nav on their page. The login, links and footer stayed as globals.